playbooks/roles/bootstrap-etcd/tasks/main.yaml

```diff
@@ -27,6 +27,7 @@
 
 - set_fact:
     ssl_data: {"*.lain.local":"web","*.lain":"web"}
+    protected_apps: ["console", "registry"]
```

Comment:
webrouter tinydns also need  protection

Reply:
#225 initialize protected_apps with ["console", "registry", "tinydns", "sso", "webrouter", "lvault"], so I remove the logic here. @supermeng